joinmarket-webui/jam
# build(deps): update dependencies

Updates following dependencies:

```
     @types/react      ^18.2.47  →  ^18.3.2
     @types/react-dom  ^18.2.18  →  ^18.3.0
     react             ^18.2.0  →  ^18.3.1
     react-dom         ^18.2.0  →  ^18.3.1
     @emotion/react    ^11.11.3  →  ^11.11.4
     bootstrap         ^5.3.2  →    ^5.3.3
     react-router-dom  ^6.21.1  →    ^6.23.1
     @testing-library/jest-dom    ^6.2.0  →    ^6.4.5
     @testing-library/react      ^14.1.2  →   ^15.0.7
     @types/jest                ^29.5.11  →  ^29.5.12
     prettier  ^3.1.1  →  ^3.2.5
```

- Update node in github actions from v16 to v20.

### How to test
After checking out and pulling the branch, run `npm install` and verify that everything is still working as expected.
Run visual tests with `npm run dev:start` and `npm start`.
Also test with `CI=true npm test` and build with `npm run build`.
